app/src/main/java/com/keylesspalace/tusky/components/login/LoginActivity.kt

```diff
@@ -99,11 +96,6 @@ class LoginActivity : BaseActivity() {
                 .into(binding.loginLogo)
         }
 
-        preferences = getSharedPreferences(
```

Comment:
This separate SharedPreferences instance was used to store temporary data that should iddeally be stored to saveInstanceState instead of polluting the global SharedPreferences.

Reply:
🧐